### PR TITLE
JBPM-5803: Stunner - Revise Stunner toolbar

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @romartin @hasys @jsoltes would you mind review this PR?
Thanks!

**Comment by romartin** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants.properties [hunk 1]
```diff
@@ -54,21 +54,23 @@ org.kie.workbench.common.stunner.core.client.toolbox.morphInto=Convert into
...
-org.kie.workbench.common.stunner.core.client.toolbox.Redo=Redo
+org.kie.workbench.common.stunner.core.client.toolbox.Redo=Redo [Ctrl+Shift+z]
 org.kie.workbench.common.stunner.core.client.toolbox.SwitchGrid=Switch grid
-org.kie.workbench.common.stunner.core.client.toolbox.Undo=Undo
+org.kie.workbench.common.stunner.core.client.toolbox.Undo=Undo [Ctrl+z]
 org.kie.workbench.common.stunner.core.client.toolbox.Validate=Validate
-org.kie.workbench.common.stunner.core.client.toolbox.VisitGraph=Visit
+org.kie.workbench.common.stunner.core.client.toolbox.VisitGraph=Play
+
+org.kie.workbench.common.stunner.core.client.toolbox.ConfirmClearDiagram=Clearing the diagram will remove all objects from the canvas permanently. You will not be able to undo this action. Proceed with caution.*
```
Comment (domhanak) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants.properties]: The "*" at the end of this String.
I am not sure if it is a typo or if it serves as a indicator to User for more caution.
Could you please confirm? :)

Comment (wmedvede) [reply kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-core-common/src/main/resources/org/kie/workbench/common/stunner/core/resources/i18n/StunnerCoreConstants.properties]: It's my fault thanks. I'll fixt it.

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
oh @wmedvede  you finally was able to disable the editor's button, cool!! Thanks!

**Pull request merged**